### ISSUE TITLE
[AXE-CORE]: VAMC - IDs used in ARIA and labels must be unique (accordions)

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#21789]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix issue.
1. At message the accessibility specialist when issue is resolved.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement
Accessibility specialists discovered Nightwatch axe checks failing silently during a recent staging review. Multiple axe violations were discovered where IDs used in ARIA and labels were not unique. This could be especially problematic to assistive technology users. Links to pages below.

**Element location**
`.facilities_health_service.usa-accordion-bordered:nth-child(4) > .usa-unstyled-list > .vads-u-margin-bottom--1 > .usa-accordion-content[aria-hidden="true"]`
**Element source**
`<div id="health_care_service_taxonomy-88" class="usa-accordion-content" aria-hidden="true">`
**To solve this violation, you need to fix the following:**
Document has multiple elements referenced with ARIA with the same id attribute: health_care_service_taxonomy-88

## Acceptance Criteria
- [x] All instances of the error are fixed
- [x] Axe-browser plugin or axe-core CLI scans show 0 Section 508, WCAG2A or WCAG2AA violations

## Environment
* ~~https://www.va.gov/cheyenne-health-care/locations/sterling-va-clinic/~~
* https://www.va.gov/erie-health-care/locations/erie-va-medical-center/
* https://www.va.gov/lebanon-health-care/health-services/
* https://www.va.gov/lebanon-health-care/locations/schuylkill-county-va-clinic/
* https://www.va.gov/lebanon-health-care/locations/york-va-clinic/
* ~~https://www.va.gov/montana-health-care/locations/hamilton-va-clinic/~~
* ~~https://www.va.gov/wilmington-health-care/health-services/~~
* ~~https://www.va.gov/wilmington-health-care/locations/sussex-county-va-clinic/~~

## Steps to Recreate
1. Open one of the listed URLs
1. Open axe browser plugin (assumes already installed in Chrome or Firefox)
1. Confirm the `ARIA attributes must conform to valid values` is present one or more times

## Screenshots
![image](https://user-images.githubusercontent.com/55411834/112062448-94b6f080-8b25-11eb-875e-882fc0192e7f.png)

**Comment by mpelzsherman** [comment]
These violations are due to duplicate data. You can see this on the page:

![VA_Cheyenne_health_care___Sterling_VA_Clinic](https://user-images.githubusercontent.com/80267/112653836-32136c80-8e25-11eb-9c4a-b6bf0965b1d6.png)

... and in the JSON data from graphql:

![VA_Cheyenne_health_care___Sterling_VA_Clinic](https://user-images.githubusercontent.com/80267/112654625-f4631380-8e25-11eb-8ebb-91715ffe44e9.png)

We can de-dupe the data on the client side, but this may indicate an issue with the CMS data. Attn: @kevwalsh

**Comment by mpelzsherman** [comment]
According to @davidconlon this is a data issue. Attn @SgardnerPMP

**Comment by mmiddaugh** [comment]
This is due to duplicated services which need to be addressed by VAMC upgrade team. Long-term CMS will need to make it impossible to duplicate these services.

**Comment by kevwalsh** [comment]
CMS team has https://github.com/department-of-veterans-affairs/va.gov-cms/issues/4074 to prevent duplicate data like this, which has been in our "stretch" list for a few sprints. Hopefully when Vet Center work is done we can get back to this.

**Comment by 1Copenut** [comment]
@kevwalsh Thank you for the follow up. One of the VSA a11y specialists is testing with JAWS and alerted me to a situation where the wrong accordion was being opened when they pressed an accordion button. I have a hunch these two things are related.

**Comment by davidconlon** [comment]
@mmiddaugh the aria issue is a FE issue. I think we have been tracking that in a different issue?

**Comment by 1Copenut** [comment]
@mmiddaugh & @davidconlon I took another look at this issue and removed two more VAMCs from the list with duplicate accordions on the page. The following VAMCs still have duplicate content:

* https://www.va.gov/erie-health-care/locations/erie-va-medical-center/
* https://www.va.gov/lebanon-health-care/health-services/
* https://www.va.gov/lebanon-health-care/locations/schuylkill-county-va-clinic/
* https://www.va.gov/lebanon-health-care/locations/york-va-clinic/

Three have a duplicate "Geriatrics" and one has a duplicate "Extended care and rehabilitation" accordion.

![Screen Shot 2021-05-06 at 11 15 59 AM](https://user-images.githubusercontent.com/934879/117332241-35a30600-ae5d-11eb-88bd-9d788c24513b.png)

---
![Screen Shot 2021-05-06 at 11 17 02 AM](https://user-images.githubusercontent.com/934879/117332259-3a67ba00-ae5d-11eb-9163-4736fe000b37.png)

**Comment by SgardnerPMP** [comment]
All duplicates resolved, with final Northern California issue resolved and PAO provided training to not duplicate system health services again.
